app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,22 @@
+class PostsController < ApplicationController
+
+  PER_PAGE = 10
+
+  def index
+    @posts = Post.order(created_at: :desc).page(params[:page]).per(PER_PAGE)
+  end
+  
+
+  def create
+   
+    @post = Post.create!(post_params)
```

Comment:
`@post` は使用していないので定義しなくてよいですね。

```suggestion
    Post.create!(post_params)
```